tests/utils.go

```diff
@@ -3850,6 +3850,22 @@ func SkipIfOpenShift4(message string) {
 	}
 }
 
+func SkipIfMigrationIsNotPossible() {
```

Comment:
Should be used [here](https://github.com/kubevirt/kubevirt/blob/d6f4ff68461474c7889a8c601e8efe18a4e2f8fd/tests/vmi_multus_test.go) as well.

Reply:
Addressed.